evap/static/ts/src/sortable-form.ts

```diff
@@ -0,0 +1,105 @@
+                // so that the one extra row doesn't change its initial value
+                (tableRow.querySelectorAll("input[id$=-order]") as NodeListOf<HTMLInputElement>).forEach(input => {
+                    input.value = "-1";
+                });
+            }
+        });
+    }
+
+    // This is the only remaining jQuery usage, since formset requires jQuery
+    $(`#${tableId} tbody tr`).formset({
+        prefix: prefix,
+        deleteCssClass: removeAsButton ? "btn btn-danger btn-sm" : "delete-row",
+        deleteText: removeAsButton ? '<span class="fas fa-trash"></span>' : window.gettext("Delete"),
+        addText: window.gettext("add another"),
+        added: function (rowJQuery: JQuery<HTMLTableRowElement>) {
+            const row = rowJQuery.get()[0];
+            (row.querySelectorAll("input[id$=-order]") as NodeListOf<HTMLInputElement>).forEach(input => {
+                input.value = row.parentElement?.childElementCount.toString() ?? "";
+            });
+
+            // We have to empty the formset, otherwise sometimes old contents from
+            // invalid forms are copied (#644).
+            // Checkboxes with 'data-keep' need to stay checked.
+            row.querySelectorAll("input[type=checkbox]:not([data-keep]),input[type=radio]").forEach(el => {
+                el.removeAttribute("checked");
+            });
+
+            (row.querySelectorAll("input[type=text],input[type=textarea]") as NodeListOf<HTMLInputElement>).forEach(
+                input => {
+                    input.value = "";
+                },
+            );
+
+            row.querySelectorAll("select").forEach(el => {
+                el.querySelectorAll("option[selected]").forEach(option => {
+                    option.removeAttribute("selected");
+                });
+                el.querySelector("option")?.setAttribute("selected", "selected");
+            });
+
+            //Check the first item in every button group
+            row.querySelectorAll(".btn-group").forEach(group => {
+                const input = group.querySelector("input");
+                if (input) {
+                    input.checked = true;
+                }
+            });
+
+            //Remove all error messages
+            row.querySelectorAll(".error-label").forEach(el => {
+                el.remove();
+            });
+
+            rowAdded(row);
+        },
+        formTemplate: usesTemplate ? ".form-template" : null,
+    });
+
+    const tableBody = document.querySelector(`#${tableId} tbody`) as HTMLElement | null;
+    if (tableBody) {
+        new Sortable(tableBody, {
+            draggable: ".sortable",
+            handle: ".fa-up-down",
+            scrollSensitivity: 70,
+        });
+
+        document.querySelectorAll("form").forEach(form => {
+            form.addEventListener("submit", () => {
+                applyOrdering();
+                return true;
```

Comment:
I think the return of `true` does nothing, only `return false` would be equivalent (but inferior) to `event.preventDefault()`